JAICore/jaicore-basic/src/main/java/jaicore/basic/ListHelper.java

```diff
@@ -5,36 +5,36 @@
 
 public class ListHelper {
```

Comment:
This class looks admittedly strange to me. I think that this should be covered in SetUtil modulo renaming. Do you really want/need this for an `Iterable`?

Reply:
Methods moved to SetUtil + Restricted Iterable to Collection.